src/components/PartyGrouping.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import Chart from 'react-apexcharts';
+import _ from 'lodash';
+
+const PartyGrouping = ({ initialTransactions }) => {
```

Comment:
If worked on a similar function for the categorization. I would suggest to have the partyGrouping done and stored on the app state level so we avoid running this logic every time the chart is rendered

Reply:
Yeah was thinking about that also. Will leave it as it is and create an issue for it